application/CohortManager/src/Functions/Shared/Common/EnumHelper.cs

```
@@ -11,19 +11,24 @@ public static class EnumHelper
     /// </summary>
     public static string GetDisplayName(Enum enumValue)
     {
-        var displayName = enumValue.GetType()
```

There may be a nicer way to handle this. Just been looking at this:

https://learn.microsoft.com/en-us/dotnet/api/system.type.gettype?view=net-9.0

There is an overload that takes a boolean argument to suppress the exception if it does not find the type.

---

Went over with Andrew but didn't seem to work for our use case